benchexec/tools/frama-c-sv.py

```diff
@@ -0,0 +1,38 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.util as util
+import benchexec.tools.template
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for Frama-C with a wrapper for usage in the SVCOMP.
+    URL: https://gitlab.com/sosy-lab/software/frama-c-sv
+    """
+
+    REQUIRED_PATHS = ["."]
+    _TOOL_NAME = "frama-c-sv"
+    _SCRIPT_NAME = _TOOL_NAME + ".py"
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable(Tool._SCRIPT_NAME)
+
+    def version(self, executable):
+        return self._version_from_tool(executable)
+
+    def name(self):
+        return Tool._TOOL_NAME
+
+    def cmdline(self, executable, options, task, rlimits):
+        cmd = [executable, "--program"] + list(task.input_files_or_identifier)
```

Comment:
```suggestion
        cmd = [executable, "--program"] + list(task.input_files)
```
According to your comment.

Reply:
Oh yes, I looked up and studied the method definitions, but I must have forgotten to actually implement the change.